### DISCUSSION TITLE
X.com suggestion (potentially out of scope)

### DISCUSSION THREAD

**Discussion by gamer191** [uBlockOrigin/uAssets#23735] [General]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://x.com/home
```

### Description

Twitter have broken Firefox's enhanced tracking protection (you'll see why I'm reporting this to uBlock Origin in a second), and now instruct users to disable it (screenshot: https://www.reddit.com/r/firefox/comments/1ctwrxe/huh_first_time_seeing_this/). They could easily fix it by temporarily stopping redirecting twitter.com to x.com, but instead they're intentionally leaving it broken to make Firefox look bad and encourage users to stop using Enhanced Tracking Protection.

Since lots of Firefox users use uBlock, I would like to propose for uBlock to block the red "Firefox’s Enhanced Tracking Protection (Strict Mode) is known to cause issues on x.com" warning. This is an anti-adblock message, so I think it is within scope to block it. Blocking it would have the following effect:

1. make Twitter look bad for breaking Firefox support, instead of letting them deflect blame to Mozilla
2. discourage users from disabling Firefox's tracking protection (you'd be surprised how many people responded to that Reddit post saying that they disabled tracing protection browser-wide)
3. strongly encourage Twitter to disable the redirect to x.com (which they temporarily disabled earlier today before they added that message)

### Other extensions used

Not really applicable imo since every Firefox user has this error. Regardless, the relevant extensions installed are ClearURLs, DuckDuckGo Privacy Essentials, Facebook Container, and User-Agent Switcher and Manager (which is set on whitelist mode, so it shouldn't be doing anything)

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.57.2
Firefox: 115
filterset (summary):
 network: 134904
 cosmetic: 46915
 scriptlet: 19470
 html: 1804
listset (total-discarded, last-updated):
 default:
  user-filters: 4-0, never
  ublock-filters: 38218-110, 6h.46m Δ
  ublock-badware: 8573-0, 6h.46m Δ
  ublock-privacy: 908-2, 6h.46m Δ
  ublock-unbreak: 2332-2, 6h.46m Δ
  ublock-quick-fixes: 195-5, 6h.46m Δ
  easylist: 87464-160, 6h.46m Δ
  easyprivacy: 50918-67, 6h.46m Δ
  urlhaus-1: 12410-3, 22h.28m
  plowe-0: 3729-1160, 3d.7h.37m
filterset (user): [array of 4 redacted]
trustedset:
 added: [array of 19 redacted]
switchRuleset:
 added: [array of 5 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1548 ms (selfie)
 maxAssetCacheWait: 111 ms
 cacheBackend: indexedDB

```
</details>

**Comment by uBlock-user** [comment]
This is what I did -- 

Set the following props to following values as below, in `about:config` --<html><body>


**urlclassifier.features.socialtracking.skipURLs**          |             ***.twimg.com, twitter.com**
  -- | --
**urlclassifier.trackingSkipURLs**                                |             ***.twimg.com, twitter.com** 



</body>
</html>
